src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/furtherevidence/actionfurtherevidence/ActionFurtherEvidenceAboutToStartHandler.java

```diff
@@ -47,7 +47,7 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
 
     private void setFurtherEvidenceActionDropdown(SscsCaseData sscsCaseData) {
         List<DynamicListItem> listOptions = getFurtherActionEvidenceItems(List.of(values()));
-        sscsCaseData.setFurtherEvidenceAction(new DynamicList(listOptions.get(0), listOptions));
+        sscsCaseData.setFurtherEvidenceAction(new DynamicList(new DynamicListItem("", ""), listOptions));
```

Comment:
could you please check if we can make the same change for [line 63](https://github.com/hmcts/sscs-tribunals-case-api/pull/3685/files#diff-866b0e8fdde319fd3fa20aed4bb81f1e50a6b0ef6d45c5fc216f920d0d55ff30L63)